File: Practice/08/C++/calculator/calculator.cpp
#include <iostream>
using namespace std; 
int main() {
	setlocale(LC_ALL, "Russian");
	char operation;
	double a, b, result;
	while (true) {
		cout << "Введите запись вычислительного процесса" << endl;
		cin >> a >> operation >> b;
		if (operation == '+') {
			result = a + b;
			cout << result << endl;
		}
		if (operation == '-') {
			result = a - b;
			cout << result << endl;
		}
		if (operation == '/') {
			if (b == 0) {
				cout << "Деление на ноль запрещено" << endl;
			}
			else {
				result = a / b;
				cout << result << endl;
			}
		}
		if (operation == '*') {
			result = a * b;
			cout << result << endl;
		}
		int answer;
		cout << "Вычислить снова: 1 - да, 0 - нет" << endl;
		cin >> answer;
		if (answer == 0) break;
	}
}

File: Practice/08/Python/calculator/calculator.py
print ("Введите запись вычислительного процесса")
variable = input().split()
a = int(variable[0])
operation = variable[1]
b = int(variable[2])
if operation == '+':
	result = a + b
	print (result)
elif operation == '-': 
	result = a - b;
	print(result)
elif operation == '/':
	if (b == 0):
		print ("Деление на ноль запрещено")
	else:
		result = a / b;
		print(result)
if operation == '*': 
	result = a * b;
	print(result)